src/js/providers/flash.js

```diff
@@ -278,6 +279,10 @@ define([
                         if (state === states.IDLE) {
                             return;
                         }
+                        if (_flashThrottledState) {
+                            _flashThrottledState = e.newstate;
+                            return;
```

Comment:
Don't forward state events while throttled (because it will clear the "error" state needed to show the Flash Blocked state).